common/src/test/java/com/revenuecat/purchases/common/OfferingsTest.kt

```diff
@@ -193,43 +229,114 @@ class OfferingsTest {
         JSONObject(
             "{'offerings': [" +
                 "${getOfferingJSON(
-                    "offering_a",
-                    PackageType.SIX_MONTH.identifier!!,
-                    "com.myproduct.sixMonth"
+                    offeringIdentifier = "offering_a",
+                    packageIdentifier = PackageType.SIX_MONTH.identifier!!,
+                    productIdentifier = "com.myproduct.sixMonth",
+                    productGroupIdentifier = "com.myproduct"
                 )}, " +
                 "${getOfferingJSON(
-                    "offering_b",
-                    PackageType.MONTHLY.identifier!!,
-                    "com.myproduct.monthly"
+                    offeringIdentifier = "offering_b",
+                    packageIdentifier = PackageType.MONTHLY.identifier!!,
```

Comment:
I think this can be added later but should we add tests for having multiple offerings with packages of the same duration?

Reply:
Yes, I plan on add more tests, for now I wanted to fix what's broken. I will add an empty one with your suggestion and a TODO so we don't forget